frontend/app/src/components/helper/display/icons/ChainIcon.vue

```diff
@@ -13,12 +13,14 @@ const props = withDefaults(defineProps<Props>(), {
 const { chain } = toRefs(props);
 
 const { getChainImageUrl } = useSupportedChains();
+
+const src = computed(() => get(getChainImageUrl(chain)));
```

Comment:
`const src = getChainImageUrl(chain)` is enough right?

Reply:
oh, should be since the function returns a computed value.

Reply:
done